src/main/java/org/teamresistance/frc/command/DriveToYX.java

```diff
@@ -0,0 +1,27 @@
+package org.teamresistance.frc.command;
+
+import org.strongback.command.CommandGroup;
+import org.teamresistance.frc.subsystem.drive.Drive;
+
+/**
+ * Created by shrey on 1/29/2017.
+ */
+public class DriveToYX extends CommandGroup {
```

Comment:
Do you think it's too early to put this into a class? This may make it harder to add more Commands to the ping sensor routine because you would have to add additional parameters to the class. I would consider removing this class and inlining the CommandGroup in `Robot.java`, similar to how the strafing CommandGroup is written.